spec/services/github_pull_request_service_spec.rb

```diff
@@ -1,9 +1,67 @@
 require 'rails_helper'
 
 describe GithubPullRequestService do
+  let(:token) { "blubli" }
+  let(:repository) { create(:repository) }
+
+  let(:github_prs_response) do
+    [
+      double(
+        id: 3,
+        title: "Test",
+        number: 1,
+        state: "open",
+        html_url: "http://www.gihub.com/prueba",
+        created_at: "2017-12-12 09:17:52",
+        updated_at: "2017-12-12 09:17:52",
+        closed_at: "2017-12-12 09:17:52",
+        merged_at: nil,
+        user: double(
+          id: 1,
+          login: "milla",
+          name: "Milla Jovovich",
+          email: "milla@jovovich.cl",
+          avatar_url: "https://avatars2.githubusercontent.com/u/741483?v=4",
+          html_url: "https://github.com/bunzli"
+        )
+      )
+    ]
+  end
+
+  let(:client) { double(:client, pull_requests: true) }
+
   def build(*_args)
     described_class.new(*_args)
   end
 
-  pending "describe what your service does here"
+  before do
+    allow(BuildOctokitClient).to receive(:for).with(token: token).and_return(client)
+
+    allow(client).to receive(:pull_requests).with(repository.full_name, state: "all")
+                                            .and_return(github_prs_response)
+  end
+
+  describe "#import_all_from_repository" do
+    context "when PR doesn`t exist" do
+      it "creates new pull request" do
+        expect { build(token: token).import_all_from_repository(repository) }.to(
+          change { PullRequest.count }
+            .from(0)
```

Comment:
Para no depender de estado anterior de la BD antes del test (si la tabla de pull request no está vacía), se podría cambiar `.from(0).to(1)` por `.by(1)`.

Reply:
ok!